plugins/pyenv/pyenv.plugin.zsh

```diff
@@ -29,6 +29,8 @@ if [[ $FOUND_PYENV -ne 1 ]]; then
 fi
 
 if [[ $FOUND_PYENV -eq 1 ]]; then
+    PYENV_VERSION="$(pyenv --version | cut -d ' ' -f 2)"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏If a variable is set, unset it at the bottom of the plugin on this line:
https://github.com/ohmyzsh/ohmyzsh/pull/9909/files#diff-b6865657813739f9d07639d70642974ef35ff2b72529eefa76e1b9bc08852d84L46

With the current behavior, this breaks the value displayed in the prompt for some reason.